### PR TITLE
add payment UI

### PR DESCRIPTION
**Pull request by Zanzan666**

### CODE REVIEW

File: web/src/pages/Payment/index.jsx [hunk 1]
```diff
@@ -1,5 +1,93 @@
...
+                      <label for="payment" class="form-label">Amount paying</label>
+                      <input type="number" min="0.00" step="0.01" max="25000" required></input>
+                      <div class="invalid-feedback">
+                        Please enter a valid amount of payment
+                      </div>
+                    </div>
+
+                    
+
+
```
Comment (wl328Weiminli) [inline web/src/pages/Payment/index.jsx:80]:
```suggestion 

```

File: web/src/pages/Payment/index.jsx [hunk 2]
```diff
@@ -1,5 +1,93 @@
...
+                    <span class="text-muted">$10</span>
+                  </li>
+                  
+                  <li class="list-group-item d-flex justify-content-between">
+                    <span>Total (USD)</span>
+                    <strong>$20</strong>
+                  </li>
+                </ul>
+        
+
```
Comment (wl328Weiminli) [inline web/src/pages/Payment/index.jsx:47]:
```suggestion
        
```

File: web/src/pages/Payment/index.jsx [hunk 3]
```diff
@@ -1,5 +1,93 @@
...
+        
+                    <div class="col-sm-6">
+                      <label for="lastName" class="form-label">Last name</label>
+                      <input type="text" class="form-control" id="lastName" placeholder="" value="" required></input>
+                      <div class="invalid-feedback">
+                        Valid last name is required.
+                      </div>
+                    </div>
+        
+
```
Comment (wl328Weiminli) [inline web/src/pages/Payment/index.jsx:69]:
```suggestion
        
```

File: web/src/index.js [hunk 4]
```diff
@@ -7,6 +7,9 @@ import "./index.css";
 import App from "./App";
 import reportWebVitals from "./reportWebVitals";
 
+import 'bootstrap/dist/css/bootstrap.css';
+import 'bootstrap/dist/js/bootstrap.min.js';
```
Comment (wl328Weiminli) [inline web/src/index.js]:
```suggestion
import 'bootstrap/dist/css/bootstrap.css';
import 'bootstrap/dist/js/bootstrap.min.js';
import App from "./App";
import reportWebVitals from "./reportWebVitals";
```

Comment (wl328Weiminli) [reply web/src/index.js]: @Zanzan666  😠 Third-party package should put before our component

**Review by yijia-cc — CHANGES_REQUESTED** [review]
Hi @Zanzan666, thank you so much for working on this feature. Please address @wl328Weiminli 's comments.

**Pull request closed without merging**